usercp.php

```diff
@@ -3221,15 +3221,18 @@
 
 			$db->insert_query("joinrequests", $joinrequest);
 
-			foreach($groupleaders[$usergroup['gid']] as $leader)
+			if(in_array($usergroup['gid'], $groupleaders))
```

Comment:
Never mind that, I was aiming for `array_key_exists()` instead; if they key exists, it's an array as defined earlier.